src/app/genes/gene-search/gene-search.component.scss

```diff
@@ -117,14 +117,15 @@
     margin-right: 30px;
 }
 
+.filter-select-error,
 .filter-select-list {
     cursor: pointer;
     padding: 5px 10px;
-    height: 60px;
+    height: auto;
```

Comment:
What's the height if you do not set it to auto here?  Is it not auto?

Reply:
It is! Good catch, I've removed the unnecessary height attribute.